CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # VocalFusion AVS Setup Change Log
 
+## 3.2.1
```

Comment:
I would use version 3.3.0 as this release will add support for a new product, so it is not a patch release.

Reply:
Hi, I considered that but it does not add a new product - XVF3800 is not part of AVS and has not been added to the setup or docs. Its bumping the dependent repo only. Makes sense?